src/clj/rems/middleware.clj

```diff
@@ -73,6 +74,7 @@
                                (when context/*user*
                                  (set/union (roles/get-roles (getx-user-id))
                                             (organizations/get-all-organization-roles (getx-user-id))
+                                            (workflows/get-all-workflow-roles (getx-user-id))
                                             (applications/get-all-application-roles (getx-user-id))))
```

Comment:
some redundancy here but I guess it won't matter, and we need to track the per-application-roles anyway for application permissions

Reply:
Yes I think that's fine.